src/components/ErrorMessage.tsx

```diff
@@ -25,7 +25,7 @@ const ErrorTitle = styled.Text`
   color: ${props => props.theme.colors.primary};
   text-align: center;
 `
-const ErrorText = styled.Text<{ centered?: boolean }>`
+export const ErrorText = styled.Text<{ centered?: boolean }>`
```

Comment:
```suggestion
export const ErrorText = styled(Content)<{ centered?: boolean }>`
```
you could use existing styled text components and remove font-size and font-family in styles here to be convenient with other Text components

Reply:
Thanks for the suggestion, that definitely makes sense.